README.md

```diff
@@ -145,6 +145,18 @@ func case5(x int) {                                //-|
 | —format | Format of the diff coverage report, one of: html, json, markdown |
 | —excludes | Exclude files for diff coverage inspection |
 
+## Miscellaneous
+
+### About `_test.go`
+
+`gocover` highly relies on cover profiles generated by `go test`. `go test` generates cover profiles for a specific package at circumstance that there is at least one `_test.go` in the package.
```

Comment:
To help reader to understand what's the "cover profiles" / "package" in this context, it would be better to provide an example and data